xrootd/encoder/encoder_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package encoder // import "go-hep.org/x/hep/xrootd/encoder"
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
I must admit I am not a huge fan of `testify/assert`.
could we just use `testing` and "inline" code to express what we expect?